relay-event-normalization/src/transactions/processor.rs

```diff
@@ -210,8 +138,10 @@ impl Processor for TransactionsProcessor<'_> {
 
         set_default_transaction_source(event);
         self.normalize_transaction_name(event);
-        self.validate_transaction(event)?;
-        end_all_spans(event)?;
+        end_all_spans(event);
+        if let Some(trace_context) = event.context_mut::<TraceContext>() {
+            trace_context.op.get_or_insert_with(|| "default".to_owned());
+        }
```

Comment:
This was the only non-validation step in the old `validate_transaction`, so we keep it here.